src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+import { Tab } from '../Types/Tab';
+
+interface Props {
+  tabs: Tab[]
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void
+}
+
+export const Tabs: FC<Props> = (props) => {
+  const { tabs, selectedTabId, onTabSelected } = props;
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
you have the same logic in App.tsx, create a helper function to find selectedTab